qualang_tools/bakery/randomized_benchmark_c1.py

```diff
@@ -29,31 +29,32 @@
 
 # Cayley table corresponding to above Clifford group structure
 
-c1_table = np.array(

```

Comment:
Please also leave the variable `c1_table = get_c1_table()`.
Otherwise you break backwards compatibility